src/app/static/src/tests/components/modelOutput/modelOutput.test.ts

```diff
@@ -264,4 +266,93 @@ describe("ModelOutput component", () => {
         bubble.vm.$emit("updateColourScales", bubbleColourScales);
         expect(store.state.plottingSelections.colourScales.output).toBe(bubbleColourScales);
     });
-});
\ No newline at end of file
+
+    it("renders choropleth table", () => {
+        const store = getStore();
+        const wrapper = shallowMount(ModelOutput, {localVue, store});
+
+        const table = wrapper.find("table-view-stub");
+        expect(table.props().areaFilterId).toBe("area");
+        expect(table.props().filters).toStrictEqual(["TEST CHORO FILTERS"]);
+        expect(table.props().selections).toStrictEqual({test: "TEST CHORO SELECTIONS"});
+        expect(table.props().indicators).toStrictEqual(["TEST CHORO INDICATORS"]);
+        expect(table.props().tabledata).toStrictEqual(["TEST DATA"]);
+    });
+
+    it("renders choropleth table with correct indicator props", () => {
+        const store = getStore({}, {
+            choroplethIndicators: jest.fn().mockReturnValue([ 
+                { "indicator": "prevalence", "indicator_value": "2" }, 
+                { "indicator": "art_coverage", "indicator_value": "4" }
+     ])}, {
+        outputChoropleth: {indicatorId: "art_coverage"}
+    });
+        const wrapper = shallowMount(ModelOutput, {localVue, store});
+
+        const table = wrapper.find("table-view-stub");
+        expect(table.props().selections).toStrictEqual({indicatorId: "art_coverage"});
+        expect(table.props().indicators).toStrictEqual([ { "indicator": "art_coverage", "indicator_value": "4" } ]);
+    });
+
+    it("renders bubble plot table", () => {
+        const store = getStore({selectedTab: "bubble"});
+        const wrapper = shallowMount(ModelOutput, {localVue, store});
+
+        const table = wrapper.findAll("table-view-stub").at(1);
+        expect(table.props().areaFilterId).toBe("area");
+        expect(table.props().filters).toStrictEqual(["TEST BUBBLE FILTERS"]);
+        expect(table.props().selections).toStrictEqual({test: "TEST BUBBLE SELECTIONS"});
+        expect(table.props().indicators).toStrictEqual(["TEST BUBBLE INDICATORS", "TEST BUBBLE INDICATORS"]);
+        expect(table.props().tabledata).toStrictEqual(["TEST DATA"]);
+    });
+
+    it("renders bubble plot table with correct indicator props", () => {
+        const store = getStore({selectedTab: "bubble"}, {
+            bubblePlotIndicators: jest.fn().mockReturnValue([ 
+                { "indicator": "prevalence", "indicator_value": "2" }, 
+                { "indicator": "art_coverage", "indicator_value": "4" },
+                { "indicator": "current_art", "indicator_value": "6" }
+     ])}, {
+        bubble: {colorIndicatorId: "art_coverage", sizeIndicatorId: "current_art"}
+    });
```

Comment:
This is a little hard to read and in general a bit of formatting can make a big difference to readability. Contrast
```
const store = getStore({selectedTab: "bubble"}, {
            bubblePlotIndicators: jest.fn().mockReturnValue([ 
                { "indicator": "prevalence", "indicator_value": "2" }, 
                { "indicator": "art_coverage", "indicator_value": "4" },
                { "indicator": "current_art", "indicator_value": "6" }
     ])}, {
        bubble: {colorIndicatorId: "art_coverage", sizeIndicatorId: "current_art"}
    });
 ```

with
```
 const store = getStore(
            {
                selectedTab: "bubble"
            },
            {
                bubblePlotIndicators: jest.fn().mockReturnValue([
                        {"indicator": "prevalence", "indicator_value": "2"},
                        {"indicator": "art_coverage", "indicator_value": "4"},
                        {"indicator": "current_art", "indicator_value": "6"}
                    ]
                )
            },
            {
                bubble: {colorIndicatorId: "art_coverage", sizeIndicatorId: "current_art"}
            });
```

Also I recommend making liberal use of IDE auto-formatting (in Intellij `ctrl-alt-l`) which will sort out some of the whitespace and alignment issues here.